auth/README.md

```diff
@@ -246,6 +272,8 @@ startActivityForResult(
     RC_SIGN_IN);
 ```
 
+###### Smart Lock hints
```

Comment:
What is a proper name? `Smart Lock` or `SmartLock` as you have below?

Reply:
Thanks! Google calls it `Smart Lock`, will fix.